test/User-test.js

```diff
@@ -207,13 +208,20 @@ describe('User', () => {
     expect(firstUser.filterFavoritesByTag('dinner')).to.deep.equal([sampleData[8]]);
   })
 
-//   it('should be able to filter favorite recipes by name', () => {
-//     firstUser.addToFavorites(sampleData[0]);
-//     firstUser.addToFavorites(sampleData[5]);
-//     firstUser.addToFavorites(sampleData[8]);
+  it('should be able to filter favorite recipes by recipe name', () => {
```

Comment:
Great test. Love that you called the function three times to check for the single instance of that name existing.